requires.py

```diff
@@ -86,3 +86,11 @@ def vault_url(self):
         :returns vault_url: URL to access vault
         :rtype vault_url: str"""
         return self.all_joined_units.received.get('vault_url')
+
+    @property
+    def vault_ca(self):
+        """Retrieve the CA published by Vault
+
+        :returns vault_ca: Base64 encoded CA Certificate data
```

Comment:
I would say the fact that the cert is b64 encoded, when being transmitted along the relation, is an implementation detail that the consumer of the interface should be abstracted from. So, I would suggest this method should return the decoded cert.

Reply:
Good point, update follows shortly